### PR TITLE
setup-project-pullrequest

### LINKED ISSUE #1: setup-project
### Issue: setup-project
- We are going to build a Leaderboard list app
- For it: we are going to install webpack
- We are going to create a folder named modules to store ES6 JS files
- We are going to create an html file to give the code's structure according to the wireframe.

### Goal:
The goal of the project is to understand ES6 and use it, as well as to feel comfortable making the set up of webpack.

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About this project:
- This is project is about a Leaderboard list app.
- We have created 2 branches : 1 named development branch (which according to the requirements is set to default)
   and the feature branch named setup-project.
- For it we have set up : Webpack, Babel compiler js, Linters.
- We have used JS -E6 using import and export keywords.
- We have created an index.html file to give the structure of the page.
- We have created JS files to add the functionality of the page.
- We have also added some config files for babel and webpack
-  We have used addEventListeners, DOM manipulation, conditionals, created classes, and created functions.

### The Goal:
The goal is to understand the usage and importance of JS to give the dynamics of a site, created new html elements with JS, how to feel comfortable with the set-up of module bundler like Webpack, JS compiler set-up like Babel.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* *{
```
Comment (t-yanick) [inline src/style.css]: - [ ] Great job working on this milestone. Kindly remove all commented code from your codebase. It makes it look professional.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import PlayerCollection from './js/app.js';
+import AppPlayerCollection from './js/display.js';
+import './style.css';
+
+// const titlePage = document.querySelector('h1');
```
Comment (t-yanick) [inline src/index.js]: - [ ] Great job working on this milestone. Kindly remove all commented code from your codebase. It makes it look professional.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @IvonneBenitesRodriguez , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Correct use of Gitflow
:heavy_check_mark: All webpack related packages should be included in package.json
:heavy_check_mark: App has basic project structure

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply src/style.css]: Hello @t-yanick 👋🏾
Very nice to meet you, 
thank you for your suggestions,
 I appreciate it, I already modified the elements you suggested me.
 Best regards,

File: src/index.js (hunk 2 above)
Comment (IvonneBenitesRodriguez) [reply src/index.js]: Hello @t-yanick 👋🏾
Very nice to meet you,
thank you for your suggestions,
I appreciate it, I already modified the elements you suggested me.
Best regards,

**Review by Reem-lab — APPROVED** [review]
# Hi @IvonneBenitesRodriguez,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descriptive PR⭐
- [ ]  Kudos for working on the setup🤩
- [ ]  No Linters error ✅
- [ ] Good job working on the requested changes ✨




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**